src/index.js

```diff
@@ -12,13 +12,19 @@ import Game from './js/game';
 $(document).ready( () => {
   const game = new Game();
   $('.start-btn').click( () => {
-    game.startRound();
+    if ($('#p1Name').val() !== '' && $('#p2Name').val() !== '' && $('#p3Name').val() !== '') {
```

Comment:
Oh nice thinking on this conditional here! That way we need them to enter a name! Nice thinking! Maybe we could have the else default their names to being something related to the game and avatars!